test/integration/user_checks_out_from_cart_test.rb

```diff
@@ -16,6 +16,12 @@ class UserChecksOutFromCartTest < ActionDispatch::IntegrationTest
       click_button "Login"
     end
 
+    StripeMock.start
+    customer = Stripe::Customer.create({
+      email: user.email_address,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.